src/middleware/index.ts

```diff
@@ -14,3 +14,13 @@ export const handleBodyRequestParsing = (router: Router) => {
 export const handleCompression = (router: Router) => {
   router.use(compression());
 };
+
+export const logErrors = (err:Error, req: Request, res: Response, next: NextFunction ) => {
+    const errStr = `ERROR url: ${req.url}\n      stack: ${err.stack}\n      message: ${err.message}`;
+    console.log(errStr);
```

Comment:
@mebassett shouldn't this one be a `console.error` https://developer.mozilla.org/en-US/docs/Web/API/Console/error

Reply:
Maybe, I'll need to double check.  PM2 is reading STDOUT for the logs.  it might be reading STDERR, too.  but console.err goes to STDERR, and I liked "PM2 will certainly log this" better than "PM2 might log this".  of course, the better solution is to check what PM2 logs.  will address in a future PR.